packages/mds-geography/package.json

```diff
@@ -14,7 +14,7 @@
     "@mds-core/mds-utils": "0.1.26",
     "express": "4.17.1",
     "sinon": "7.5.0",
-    "uuid": "7.0.1"
+    "uuid": "7.0.2"
```

Comment:
Surprised this didn't update the `yarn.lock` file

Reply:
All the other packages are on 7.0.2.